.gitignore

```diff
@@ -40,4 +40,9 @@ launch.json
 
 # profiling files
 chrome-profiler-events*.json
-speed-measure-plugin*.json
\ No newline at end of file
+speed-measure-plugin*.json
+
+#sassdoc files generated on build
+sass-vars.json
```

Comment:
if we're adding files to the gitignore, you should delete them from the repo also